### PR TITLE
Convert header to react with pf's components

### PR DESCRIPTION
**Pull request by karelhala**

### CODE REVIEW

**Comment by karelhala** [comment]
>  the icons (settings, search, etc) on mobile are really hard to see.
That's because they are disabled, I am gonna change them so when on large screen they will be icons and smaller screens they will be part of the navigation as text, preferably secondary dropdown (when we will know what to show there)

**Comment by ryelo** [comment]
@karelhala you've got a few unused vars that is failing the build

File: src/js/redux/action-types.js [hunk 1]
```diff
@@ -1,3 +1,5 @@
+export const TOGGLE_ACTION = '@@chrome/toggle';
```
Comment (jharting) [inline src/js/redux/action-types.js]: &lt;nitpick&gt;This is quite ambiguous as it is not clear which toggle this is. Might be helpful to be more specific

File: src/js/App/Header/Brand.js [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+            <BarsIcon size="md"/>
+        </Button>
+    </div>
+    <a className="pf-l-page__header-brand-link" href="./platform/dashboard/">
+        <Logo />
+    </a>
+</div>);
+
+function mapDispatchToProps(dispatch) {
+    return { toggleNav: () => {
```
Comment (jharting) [inline src/js/App/Header/Brand.js]: &lt;nitpick&gt;weird indentation

**Pull request merged**